mod-orders-storage/schemas/po_number.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Generated PO number object",
+  "type": "object",
+  "properties": {
+    "po_number": {
+      "description": "generated purchase order number",
+      "type": "string",
+      "pattern": "^[1-9][0-9]{4,15}$"
```

Comment:
I think since this schema is likely to be reused by the validate endpoint we should just make the pattern more generic... `^[0-9]{5-16}$`